src/multiparser.js

```diff
@@ -46,12 +46,14 @@ function fromBabylonFlowOrTypeScript(path) {
       if (isCss) {
         // Get full template literal with expressions replaced by placeholders
         const rawQuasis = node.quasis.map(q => q.value.raw);
-        const text = rawQuasis.join("@prettier-placeholder");
-        return {
-          options: { parser: "css" },
-          transformDoc: transformCssDoc,
-          text: text
-        };
+        if (rawQuasis.some(q => q)) {
```

Comment:
In `` `${a}${b}${c}` `` all quasis will be the empty string (falsy), right? What happens then?

Reply:
Yeah it prints `${a}${b}${c}` even without this change.

Let me try to fix it!

Reply:
@lydell I'm not sure what we should do on this case: [playground](https://prettier.io/playground/#%7B%22content%22%3A%22const%20a%20%3D%20styled.div%60%24%7Ba%7D%24%7Bb%7D%24%7Bc%7D%60%3B%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)

Reply:
I don't really care about that case, I was just confused by this `if` statement. I like your new commit where you explicitly check for a completely empty template literal. :+1: